components/mintmaker/development/kustomization.yaml

```diff
@@ -16,3 +16,6 @@ commonAnnotations:
 
 components:
   - ../components/rh-certs
+
+patches:
+  - path: renovate-config.yaml
```

Comment:
For development env, we can just use "components/mintmaker/base/renovate-config.yaml".